versions.js

```diff
@@ -0,0 +1,60 @@
+
+'use strict';
+
+var versions;
+
+//  declare the current versions for the build system to use
+versions =
+[
+  {
+    "name": "io.js",
+    "nameRegExp": /{{\s*io\.js_latest\.name\s*}}/mg,
+    "value": "1.3.0",
+    "valueRegExp": /{{\s*io\.js_latest\.value\s*}}/mg,
+    "link": "https://iojs.org/dist/v1.3.0/",
+    "linkRegExp": /{{\s*io\.js_latest\.link\s*}}/mg,
+  },
+  {
+    "name": "Linux",
+    "nameRegExp": /{{\s*io\.js_latest_linux\.name\s*}}/mg,
```

Comment:
Will change that.

I thought of having a convention of how to construct these RegExps, but I wasn't sure until I see all the use cases we'll have.